services/minespace-web/tsconfig.json

```diff
@@ -17,6 +17,7 @@
     "esModuleInterop": true /* Emit additional JavaScript to ease support for importing CommonJS modules. This enables 'allowSyntheticDefaultImports' for type compatibility. */,
     "forceConsistentCasingInFileNames": true /* Ensure that casing is correct in imports. */,
     "strict": false /* Enable all strict type-checking options. */,
+    "strictNullChecks": false,
```

Comment:
hmm, we do have significant bugs that come up because of missed null checks- was this to prevent TS errors about all the pre-existing cases? I think if/when we merge this we'll want to make a ticket to remove the line and fix all the errors.

Reply:
Actually, good catch.  That had been added in at one point in the process to test something and I forgot to remove it.  I'll take it out.